src/agent.ts

```diff
@@ -290,70 +290,70 @@ export class Agent {
   }
 
   /**
-   * Creates Agent as string using agent job spec data.
+   * Creates AgentScriptContent using agent job spec data.
    *
    * @param connection The connection to the org
    * @param agentJobSpec The agent specification data
-   * @returns Promise<AgentString> The generated Agent as a string
+   * @returns Promise<AgentScriptContent> The generated AgentScriptContent
    * @beta
    */
-  public static async createAgent(connection: Connection, agentJobSpec: AgentJobSpec): Promise<AgentString> {
+  public static async createAgentScript(connection: Connection, agentJobSpec: AgentJobSpec): Promise<AgentScriptContent> {
     const url = '/connect/ai-assist/create-af-script';
     const maybeMock = new MaybeMock(connection);
 
     getLogger().debug(`Generating Agent with spec data: ${JSON.stringify(agentJobSpec)}`);
 
-    const response = await maybeMock.request<CreateAgentResponse>('POST', url, agentJobSpec);
-    if (response.isSuccess && response.agentString) {
-      return response.agentString;
+    const response = await maybeMock.request<CreateAgentScriptResponse>('POST', url, agentJobSpec);
+    if (response.isSuccess && response.agentScriptContent) {
+      return response.agentScriptContent;
     } else {
       throw SfError.create({
-        name: 'CreateAgentError',
+        name: 'CreateAgenScriptError',
         message: response.errorMessage ?? 'unknown',
         data: response,
       });
     }
   }
 
   /**
-   * Creates agent JSON from compiling the Agent as a string on the server.
+   * Creates agent JSON from compiling the AgentScriptContent on the server.
    *
    * @param connection The connection to the org
-   * @param AgentString The agent Agent as a string
-   * @returns Promise<string> The generated Agent JSON as a string
+   * @param agentScriptContent The AgentScriptContent to compile
+   * @returns Promise<AgentJson> The generated Agent JSON
    * @beta
    */
-  public static async compileAgent(connection: Connection, agentString: AgentString): Promise<AgentJson> {
+  public static async compileAgentScript(connection: Connection, agentScriptContent: AgentScriptContent): Promise<AgentJson> {
```

Comment:
This doesn't return the AgentJson type.  Agent JSON is part of the response when it's successful.  See the API docs.